examples_tests/49.Radix_Sort/compute_shader.comp

```diff
@@ -1,12 +1,42 @@
 // Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
 // This file is part of the "Nabla Engine".
 // For conditions of distribution and use, see copyright notice in nabla.h
+#version 430 core
+#define G_IDX gl_GlobalInvocationID.x
+#define G_IDY gl_GlobalInvocationID.y
+#define LC_IDX gl_LocalInvocationIndex
 
-#version 450
 
+layout(local_size_x = 1024, local_size_y = 1024) in;
 
 
+layout(std430, binding = 0) restrict readonly buffer InBuf {
+	int input[];
+};
+layout(std430, binding = 1) restrict writeonly buffer OutBuf {
+	int output[];
+};
+
+shared Temp[1024];
+
+float count_occurencies(int input)
+{
+    return 1.0;
+}
+
 void main()
 {
-	
+
+
+    for(uint i = 0; i < 1024; ++i)  //y dim
+    {   
+        for(uint j = 0; j < 1024; ++j) //x dim
+        {
```

Comment:
use `nbl_glsl_workgroupBallot` instructions

Reply:
It is temporary, but thanks